### ISSUE TITLE
channel flow DNS case broken

### ISSUE THREAD

**Issue by rmcdermo** [firemodels/fds#11726]
Saw this on a visual scan of the verification guide.  I'll work on it.

<img width="224" alt="Screen Shot 2023-04-16 at 7 55 27 AM" src="https://user-images.githubusercontent.com/4418497/232308159-c0875843-216a-468a-8eb2-7d0af4167f42.png">

**Comment by drjfloyd** [comment]
I ran the Pr=2.0 case. The k, nu, and cp in the .out file look OK for the
lumped species but notice a couple of minor things.

Gibbs is being written. I think in the case where someone overrides
anything related to enthalpy for a predefined species that we can no longer
use the predefined Gibbs as Gibbs is a function of enthalpy. This was
something I suspect was happening before my recent changes to species
procession, but until those changes we weren't writing Gibbs to the .out
file.

In the .out, the primitive species has the predefined H_F for LJ AIR, but
the lumped species has H_F = SPECIFIC_HEAT *H_F_REFERENCE_TEMPERATURE.
Here I think the two should match.

The Gibbs shouldn't be a cause of the problem for the case as it only
impacts reversible reactions. I don't thin the H_F should, but maybe?

**Comment by rmcdermo** [comment]
These seem to be running fine.  I bet there is just some hiccup in the output processing.  Once I get the latest output in a few hours I'll pin that down.

**Comment by rmcdermo** [comment]
This is an output issue.  Probably something related to using INTERPOLATION with time averaging.  For the PR=2 case, this is what Smokeview gives, but the line file has 0s for the temperature.

<img width="752" alt="Screenshot 2023-04-17 at 10 31 30 AM" src="https://user-images.githubusercontent.com/4418497/232518117-02ffabe0-06d7-40da-861c-951ee4d78606.png">

**Comment by rmcdermo** [comment]
Did a bisect.  The issue seems to be that the behavior of NET HEAT FLUX (now TOTAL HEAT FLUX) in combination with RADIATION=F changed here:
```
2cb31f86c0bd525d4a6328cd92d6e8e982d5ce54 is the first bad commit
commit 2cb31f86c0bd525d4a6328cd92d6e8e982d5ce54
Author: Jason Floyd <drjfloyd@gmail.com>
Date:   Wed Jan 18 13:55:03 2023 -0500

    FDS Source: Move ONE_D%Q_RAD_OUT from radi to wall
```

Since I have changed the input files to use CONVECTIVE HEAT FLUX, this should not really matter anymore for this case.  But the question remains: should ONE_D%Q_RAD_OUT be applied if RADIATION=F.  Up to you guys, just so long as it is documented.  And so that you know this behavior did change in the last couple months.

**Comment by drjfloyd** [comment]
This was changed as what we were reporting in output files wasn't the same radiative flux being used in wall.f90. I'll look at the source and see if we need a change in the source or in the docs.

**Comment by rmcdermo** [comment]
The users guide is not crystal clear.  It only discusses radiant loss by the fire.  That is, it focusses on whether RADIATION=F will still allow a RADIATIVE_FRACTION.

However, it seems that the behavior of the code has now changed such that if RADIATION=F, then all surfaces still emit (and absorb) radiation and that just KAPPA=0.  Given that this would apply to particles as well, this seems like a drastic change in behavior.   Suppose you just wanted to consider convective heat transfer to and from a droplet, then you would think that setting RADIATION=F would allow this to happen.

**Comment by drjfloyd** [comment]
Yep I missed that. I think we just need to add IF (RADIATION) before each time we set Q_RAD_OUT in wall and that should replicate the behavior when we used to set it in radi (if no radiation we never called the routine that set it).

**Comment by mcgratta** [comment]
The surface emit, but that energy just disappears, as does the energy from the fire. The surfaces do not absorb, since Q_RAD_IN=0.

**Comment by drjfloyd** [comment]
We never actually used Q_RAD_OUT in the 1D routine. If there was no radiation we never entered the solver branch that used it. The output value for 1D was getting set after the solver.

**Comment by rmcdermo** [comment]
Please keep in mind that this case (heated channel) has very small temperature differences.  The wall temperature is 20 C and the gas temperatures get to about 25 C.  Radiation should not matter at all here.  Yet, because of this change, it matters a lot.  Something is not right.

**Comment by drjfloyd** [comment]
I think this should be fixed now.  Before this change Q_RAD_OUT was set in radi.f90. When I moved to wall.f90 (which is done whether or not there is RADIATION), I didn't add a check for that so we were always setting Q_RAD_OUT. I went through the code and:

-anyplace we initialize Q_RAD_OUT, it is now set to zero if RADIATION=F
-anyplace we compute Q_RAD_OUT based on current conditions is only done if RADIATION=T.

This should set behavior back to like it was when Q_RAD_OUT was computed in radi where it was only updated for current conditions when RADIATION=T.
